lib/srv/reexec_linux.go

```diff
@@ -38,23 +36,20 @@ func init() {
 	// this can happen if both calls returned -1 or if we're in running in a
 	// version of qemu-user that's affected by this bug:
 	// https://gitlab.com/qemu-project/qemu/-/issues/927
+	// (hopefully they'll also add special handling for execve on /proc/self/exe
+	// if they ever fix that bug)
 	if fd1 == fd2 {
 		return
 	}
 
 	// if one has failed but not the other we can't really trust what's
 	// happening anymore
```

Comment:
Not sure if this comment is accurate here anymore. Probably should be moved below?

Reply:
That's still accurate but harder to understand; I'll reword the comment to specifically mention that it's referring to the `Open` calls.